samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -0,0 +1,366 @@
+		      struct net_buf_simple *buf)
+{
+	char le_addr[BT_ADDR_LE_STR_LEN];
+	char name[NAME_LEN];
+	int64_t time_delta_ms;
+
+	time_delta_ms = k_uptime_get() - start_time;
+
+	(void)memset(name, 0, sizeof(name));
+
+	bt_data_parse(buf, data_cb, name);
+
+	bt_addr_le_to_str(info->addr, le_addr, sizeof(le_addr));
+
+	printk("[DEVICE]: %s, AD evt type %u, Tx Pwr: %i, RSSI %i %s C:%u S:%u "
+	       "D:%u SR:%u E:%u Prim: %s, Secn: %s, Interval: 0x%04x (%u ms), "
+	       "SID: %u\n",
+	       le_addr, info->adv_type, info->tx_power, info->rssi, name,
+	       (info->adv_props & BT_GAP_ADV_PROP_CONNECTABLE) != 0,
+	       (info->adv_props & BT_GAP_ADV_PROP_SCANNABLE) != 0,
+	       (info->adv_props & BT_GAP_ADV_PROP_DIRECTED) != 0,
+	       (info->adv_props & BT_GAP_ADV_PROP_SCAN_RESPONSE) != 0,
+	       (info->adv_props & BT_GAP_ADV_PROP_EXT_ADV) != 0,
+	       phy2str(info->primary_phy), phy2str(info->secondary_phy),
+	       info->interval, info->interval * 5 / 4, info->sid);
+
+	if (!per_adv_found && info->interval) {
+
+		per_adv_found = true;
+		per_sid = info->sid;
+		bt_addr_le_copy(&per_addr, info->addr);
+
+		k_sem_give(&sem_per_adv);
+	}
+}
+
+static void create_sync(void)
+{
+	int err;
+
+	printk("Creating Periodic Advertising Sync...");
+	bt_addr_le_copy(&sync_create_param.addr, &per_addr);
+	sync_create_param.options = 0;
+	sync_create_param.sid = per_sid;
+	sync_create_param.skip = 0;
+	sync_create_param.timeout = 0xa;
+	err = bt_le_per_adv_sync_create(&sync_create_param, &sync);
+	if (err) {
+		printk("failed (err %d)\n", err);
+		return;
+	}
+	printk("success.\n");
+}
+
+static int delete_sync(void)
+{
+	int err;
+
+	printk("Deleting Periodic Advertising Sync...");
+	err = bt_le_per_adv_sync_delete(sync);
+	if (err) {
+		printk("failed (err %d)\n", err);
+		return err;
+	}
+	printk("success\n");
+	return 0;
+}
+
+static void enable_cte_rx(void)
+{
+	int err;
+
+	const struct bt_df_per_adv_sync_cte_rx_param cte_rx_params = {
+		.max_cte_count = 5,
+#if defined(CONFIG_BT_CTLR_DF_ANT_SWITCH_RX)
+		.cte_type = BT_DF_CTE_RX_TYPE_ALL,
+		.slot_durations = 0x2,
+		.num_ant_ids = ARRAY_SIZE(ant_patterns),
+		.ant_ids = ant_patterns,
+#else
+		.cte_type = BT_DF_CTE_RX_TYPE_AOD_1US | BT_DF_CTE_RX_TYPE_AOD_2US,
+#endif /* CONFIG_BT_CTLR_DF_ANT_SWITCH_RX */
+	};
+
+	printk("Enable receiving of CTE...\n");
+	err = bt_df_per_adv_sync_cte_rx_enable(sync, &cte_rx_params);
+	if (err) {
+		printk("failed (err %d)\n", err);
+		return;
+	}
+	printk("success. CTE receive enabled.\n");
+}
+
+static int scan_init(void)
+{
+	printk("Scan callbacks register...");
+	bt_le_scan_cb_register(&scan_callbacks);
+	printk("success.\n");
+
+	printk("Periodic Advertising callbacks register...");
+	bt_le_per_adv_sync_cb_register(&sync_callbacks);
+	printk("success.\n");
+
+	printk("Periodic Advertising CTE receive callbacks register...");
+	bt_df_per_adv_sync_cte_cb_register(&cte_report_callbacks);
+	printk("success.\n");
+	return 0;
```

Comment:
I probably did, but it's one of the conventions I don't feel that strongly about. Here however it does make the code look slightly nicer looking (IMO) since you've then got the three similar blocks more clearly separated.

Reply:
Empty line added.